src/main/java/org/sonarsource/sonarlint/ls/SonarLintLanguageServer.java

```diff
@@ -142,7 +150,12 @@ static SonarLintLanguageServer bySocket(int port, Collection<URL> analyzers) thr
       cancelToken.checkCanceled();
       this.traceLevel = parseTraceLevel(params.getTrace());
 
-      progressManager.setWorkDoneProgressSupportedByClient(ofNullable(params.getCapabilities().getWindow().getWorkDoneProgress()).orElse(false));
+      progressManager.setWorkDoneProgressSupportedByClient(
+        ofNullable(params.getCapabilities())
```

Comment:
Got a NPE when testing in VSCode, I believe this code is a bit defensive but at least it avoids an early crash :smile: